server/plugin/command.go

```diff
@@ -702,6 +704,107 @@ func (p *Plugin) handleIssue(_ *plugin.Context, args *model.CommandArgs, paramet
+
+	switch {
+	case command == "set":
+		return p.handleSetDefaultRepo(c, args, parameters, userInfo)
+	case command == "get":
+		return p.handleGetDefaultRepo(c, args, parameters, userInfo)
+	case command == "unset":
+		return p.handleUnSetDefaultRepo(c, args, parameters, userInfo)
+	default:
+		return fmt.Sprintf("Unknown subcommand %v", command)
+	}
+}
+
+func (p *Plugin) handleSetDefaultRepo(_ *plugin.Context, args *model.CommandArgs, parameters []string, userInfo *GitHubUserInfo) string {
+	if len(parameters) == 0 {
+		return "Please specify a repository."
+	}
+
+	repo := parameters[0]
+	config := p.getConfiguration()
+	baseURL := config.getBaseURL()
+	owner, repo := parseOwnerAndRepo(repo, baseURL)
+	if owner == "" || repo == "" {
+		return "Please provide a valid repository"
+	}
+
+	owner = strings.ToLower(owner)
+	repo = strings.ToLower(repo)
+
+	if config.GitHubOrg != "" && strings.ToLower(config.GitHubOrg) != owner {
+		return fmt.Sprintf("Repository is not part of the locked Github organization. Locked Github organization: %s", config.GitHubOrg)
+	}
+
+	ctx := context.Background()
+	githubClient := p.githubConnectUser(ctx, userInfo)
+
+	ghRepo, _, _ := githubClient.Repositories.Get(ctx, owner, repo)
+	if ghRepo == nil {
+		return fmt.Sprintf("unknown repository %s", fullNameFromOwnerAndRepo(owner, repo))
+	}
+
+	if _, err := p.store.Set(fmt.Sprintf(DefaultRepoKey, args.ChannelId, userInfo.UserID), []byte(owner+"/"+repo)); err != nil {
+		return "error occurred saving the default repo"
+	}
+
+	repoLink := baseURL + owner + "/" + repo
+	successMsg := fmt.Sprintf("The default repo has been set to [%s/%s](%s) for this channel", owner, repo, repoLink)
+
+	return successMsg
+}
+
+func (p *Plugin) GetDefaultRepo(userID string, channelID string) (string, error) {
+	var defaultRepoBytes []byte
+	if err := p.store.Get(fmt.Sprintf(DefaultRepoKey, channelID, userID), &defaultRepoBytes); err != nil {
+		return "", err
+	}
+
+	return string(defaultRepoBytes), nil
+}
+
+func (p *Plugin) handleGetDefaultRepo(_ *plugin.Context, args *model.CommandArgs, _ []string, userInfo *GitHubUserInfo) string {
+	defaultRepo, err := p.GetDefaultRepo(userInfo.UserID, args.ChannelId)
+	if err != nil {
+		p.client.Log.Warn("Not able to get the default repo", "error", err.Error())
+		return "error occurred while getting the default repo"
+	}
+
+	if defaultRepo == "" {
+		return "you have not set a default repository for this channel"
+	}
+
+	config := p.getConfiguration()
+	repoLink := config.getBaseURL() + defaultRepo
+	return fmt.Sprintf("The default repository is [%s](%s)", defaultRepo, repoLink)
+}
+
+func (p *Plugin) handleUnSetDefaultRepo(_ *plugin.Context, args *model.CommandArgs, _ []string, userInfo *GitHubUserInfo) string {
+	defaultRepo, err := p.GetDefaultRepo(userInfo.UserID, args.ChannelId)
+	if err != nil {
+		p.client.Log.Warn("Not able to get the default repo", "error", err.Error())
+		return "error occurred while getting the default repo"
```

Comment:
```suggestion
		return "Error occurred while getting the default repo"
```
I think all these messages should be capatalised